pyspecProcScripts/correlation_alignment.py

```diff
@@ -251,7 +269,4 @@ def correl_align(
             "after correlation\nph0 restored \nsig. energy=%g" % sig_energy
         )
         fl.pop_marker()
-    if avg_dim:
-        s_orig.chunk(avg_dim, [avg_dim, "power"], [avg_dim_len, -1])
-        s_orig.reorder(["ph1", avg_dim, "power", direct])
```

Comment:
we couldn't figure out what this was doing here since s_orig was never returned and this outside of the loop??

Reply:
hmm, it's possible that it's undoing things that were done to `s_orig` earlier.  Even if `s_orig` isn't returned, if you mess with its contents, they will change, so this could be undoing stuff that was done at the top.  Specifically, it looks like it's trying to undo a smoosh and reorder.

Reply:
I made the smooshed version the copy so s_orig no longer gets modified so these are not necessary

Reply:
I think that I'm very confused by this and we should sit down and review this together in the code